pandas/_libs/src/vendored/ujson/python/objToJSON.c

```diff
@@ -1532,7 +1521,6 @@ void Object_beginTypeContext(JSOBJ _obj, JSONTypeContext *tc) {
 
     PyArray_Descr *outcode = PyArray_DescrFromType(NPY_INT64);
     PyArray_CastScalarToCtype(obj, &longVal, outcode);
-    Py_DECREF(outcode);
```

Comment:
Just confirming this was also an intentional cleanup?

Reply:
No, this is an accident and shouldn't be included.

Reply:
Thanks for the catch.

I wonder if we're leaking references to PyArray_Descr elsewhere than. 

The numpy docs don't seem too clear about whether we're borrowing references or owning it.

Reply:
Ah, yeah the deleted code here used to leak them...  One point is that array creation functions steal them, though.